docs/guides/options.md

```diff
@@ -1,138 +1,352 @@
+
+## Setting Options
+
+Because Video.js decorates an HTML5 `<video>` element, many of the options available to you are also available as [standard `<video>` tag attributes][video-attrs]:
 
 ```html
 <video controls autoplay preload="auto" ...>
 ```
 
-Alternatively, you can use the data-setup attribute to provide options in the [JSON](http://json.org/example.html) format. This is also how you would set options that aren't standard to the video tag.
+Alternatively, you can use the `data-setup` attribute to provide options as [JSON][json]. This is also how you would set options that aren't standard to the `<video>` element:
 
 ```html
-<video data-setup='{ "controls": true, "autoplay": false, "preload": "auto" }'...>
+<video data-setup='{"controls": true, "autoplay": false, "preload": "auto"}'...>
+```
+
+Finally, if you're not using the `data-setup` attribute to trigger the player setup, you can pass in an object of player options as the second argument to the `videojs` function:
+
+```js
+videojs('my-player', {
+  controls: true,
+  autoplay: false,
+  preload: 'auto'
+});
 ```
 
-Finally, if you're not using the data-setup attribute to trigger the player setup, you can pass in an object with the player options as the second argument in the javascript setup function.
+### Setting Global Defaults
+
+Default options for all players can be found at `videojs.options`. In order to override them, mutate this object as needed. For example, to set `{autoplay: true}` for all players:
 
 ```js
-videojs("example_video_1", { "controls": true, "autoplay": false, "preload": "auto" });
+videojs.options.autoplay = true;
 ```
 
+### A Note on `<video>` Tag Attributes
 
-Individual Options
-------------------
+Many attributes are so-called [boolean attributes][boolean-attrs]. This means they are either on or off. In these cases, the attribute _should have no value_ - its presence implies a true value and its absence implies a false value.
 
-> ### Note on Video Tag Attributes ###
-> With HTML5 video tag attributes that can only be true or false (boolean), you simply include the attribute (no equals sign) to turn it on, or exclude it to turn it off. For example, to turn controls on:
+_These are incorrect:_
 
-WRONG
 ```html
 <video controls="true" ...>
+<video controls="false" ...>
 ```
 
-RIGHT
+> **Note:** The example with `controls="false"` can be a point of confusion for new developers - it will actually turn controls _on_!
+
+These are correct:
+
 ```html
 <video controls ...>
+<video ...>
 ```
 
-> The biggest issue people run into is trying to set these values to false using false as the value (e.g. controls="false") which actually does the opposite and sets the value to true because the attribute is still included. If you need the attribute to include an equals sign for XHTML validation, you can set the attribute's value to the same as its name (e.g. controls="controls").
+### Precedence
```

Comment:
This seems relevant, though. The sort of thing I'd want to know without having to experiment for myself. I feel like it's the implicit reality.